content/en/funding/community-projects/community-projects-process.mdx

```diff
@@ -1,124 +1,187 @@
 
 </Box>
 
 <Box>
 
-## Start work
+## Start Work
+
 If it’s a small or medium scoped project and you’ve addressed any feedback, work can then begin.
+
 If it’s a large project, a proposal will need to be split into GitHub issues/tasks and another call with the approving group will be needed to confirm costs and resources.
 
 </Box>
 
 <Box>
 
-## Communicate progress
+## Communicate Progress
 The project will have checkpoints and you will be expected to present progress at Community weekly syncs.
 
 </Box>
 
 <Box>
 
-## Evaluate at a retrospective
+## Evaluate at a Retrospective
 Once the work is done, there’ll be a review and retrospective to show off the work, discuss what went well and what can be improved.
 
 </Box>
 
 </Process>
 
-## Resources
+<CTA icon="question_mark" secondary>
 
-You should outline anything you need in your proposal. For example:
+If you are unsure about your proposal, post your idea in the chat for feedback. An Advisor will help you iterate on your proposal.
 
-- People support \(developer, designer, editor, writer, videographer, etc\)
-- Foundation resources support \(brand assets, official product screenshots, etc\)
+</CTA>
 
-If you need more resources than you first thought, let your advisor know. They will coordinate a meeting, outline the required resources, and present them to the approving group. They’ll figure out whether to approve additional resources or not.
+<Accordion>
 
-### Community resources
+## **DO**
 
-Here’s a list of resources that might be useful:
+- Check [existing or past proposals](https://github.com/makerdao/community/issues?q=label%3ACDIP+).
+- Spend 1-3 hours on properly fleshing out your proposal.
+- Bring friends and colleagues in to help if necessary.
+- Solicit feedback early and often.
+- Have some idea about when to check in with the team and your CDIP's advisor on progress or blockers.
 
-<List>
+</Accordion>
+
+<Accordion>
+
+## **DON'T**
+
+- Invoice for work before approval, you may not be paid for it.
+- Submit more than one proposal at a time.
+- Duplicate an existing proposal.
+- Submit a proposal for small pieces of work.
+
+</Accordion>
+
+## Community Resources
 
-<Link to="/work_with_us/content/writing-style-guide">
+Here’s a list of resources you may find useful when executing on your proposal:
 
-Writing style guide
+<List>
 
+[Writing style guide](/work_with_us/content/writing-style-guide) 
 How to write like a Maker contributor
+    
+[Editorial standards](/work_with_us/content/reviewer-guide) 
+ What to look out for when reviewing work
 
-</Link>
+[Visual style guide](/work_with_us/content/visual-style-guide) 
+How to look like Comm-Dev at Maker
 
+<Link to="https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4">Design files and assets</Link>Official Maker and Dai logos, badges, and product screenshots from the Foundation design team.
 
-<Link to="/work_with_us/content/reviewer-guide">
+</List>
 
-Editorial standards
+## Proposal Details
```

Comment:
here down to right before questions should be accordion. Accordians are great for "I don't have to know this information yet or I want to expand more details"